src/filesys.cpp

```diff
@@ -740,5 +740,26 @@ bool Rename(const std::string &from, const std::string &to)
 	return rename(from.c_str(), to.c_str()) == 0;
 }
 
+bool bad_char(const char c){
```

Comment:
`isBadChar` would fit better as a function name IMO.
Please put the bracket on a new line.